src/main/java/frc/robot/RobotContainer.java

```diff
@@ -63,8 +64,7 @@ public RobotContainer() {
   private void configureBindings() {
     // Joystick suppliers,
     Trigger leftShoulderTrigger = joystick.leftBumper();
-
-    Supplier<Double>
+    DoubleSupplier
```

Comment:
Java primitives have special supplier types which avoid creating a boxed type. This reduces GC pressure since the allocated memory is on the stack (since they're primitives) instead of the heap, which is easier to tear down when no longer needed.